node/pkg/vaa/structs.go

```diff
@@ -203,6 +203,10 @@ const (
 )
 
 // Unmarshal deserializes the binary representation of a VAA
+//
+// WARNING: Unmarshall will truncate payloads at 1000 bytes, this is done mainly to avoid denial of service
+//   - If you need to access the full payload, consider parsing VAA from Bytes instead of Unmarshal
+//
```

Comment:
If we are truncating stuff, does it make sense to add a log? I'm not sure.

Reply:
going to pass on this now in the primative, I'd like to make a sweep once we have better monitoring setup, and then make this more robust.